orchestration-shared/src/test/java/uk/gov/di/orchestration/shared/services/LogoutServiceTest.java

```diff
@@ -206,7 +212,7 @@ void successfullyReturnsClientLogoutResponse() {
         verify(backChannelLogoutService)
                 .sendLogoutMessage(
                         argThat(withClientId("client-id")), eq(EMAIL), eq(INTERNAL_SECTOR_URI));
-        verify(cloudwatchMetricsService).incrementLogout(Optional.of(CLIENT_ID));
+        verify(cloudwatchMetricsService).incrementLogout(Optional.of(CLIENT_ID), Optional.empty());
```

Comment:
why add this `Optional.empty()` argument? There's already a method to handle `intervention` being not Optional.empty(). Same for all the cases in this file.

Reply:
It's due to the refactor of the destroySession method. However it's really a symptom of using `verify` as we're leaking implementation details into the unit test. Ideally we'd be using a fake instead of a mock and we could query the increment value.